src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Install and configure the Prometheus agent with New Relic for Kubernetes
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus agent
+metaDescription: 'How to install, update, or uninstall your New Relic Prometheus agent with New Relic for Kubernetes.'
+---
+
+New Relic's Prometheus agent is a thin wrapper (we refer to it as Prometheus Configurator in [its repository](https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/charts/newrelic-prometheus-agent/values.yaml#L184)) on top of Prometheus to run Prometheus in agent mode and its configuration burden. With this integration you can create a YAML configuration file used by Prometheus ready to send the metrics to New Relic.
+
+## Install in Kubernetes [#kubernetes]
+
+See how to [install the Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure), which includes the feature flag for Prometheus Configurator.
+
+Alternatively, we also offer fully manual instructions for [deploying our integration using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm).
+
+By default, the New Relic Prometheus agent uses the same annotation as Prometheus to discover targets: `prometheus.io/scrape: true`. For more information, see the [set up New Relic's Prometheus agent](/docs/integrations/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent) documentation.
+
+To ensure the integration has been configured correctly, go to **[one.newrelic.com](https://one.newrelic.com)** > Query builder, and run this NRQL query to see if data has been reported:
+
+```sql
+FROM Metric SELECT count(*) WHERE clusterName = 'YOUR_CLUSTER_NAME' since 1 hour ago
```

Comment:
should we scope this with collector.name='prometheus-agent'? (in case metrics from other integrations also contain a clusterName dimension/attribute)